.gitignore

```diff
@@ -38,8 +39,4 @@
 .Spotlight-V100
 .Trashes
 ehthumbs.db
-Thumbs.db
-
-*.ini
-
-*.txt
+Thumbs.db
```

Comment:
I would add `user.ini` though. So do you don't accidentally commit it. Makes it easier when forking / creating PRs.